src/mul_base.rs

```diff
@@ -0,0 +1,25 @@
+// -*- mode: rust; -*-
+//
+// This file is part of ed25519-dalek.
+// Copyright (c) 2017-2019 isis lovecruft
+// See LICENSE for licensing information.
+
+use curve25519_dalek::constants;
+use curve25519_dalek::edwards::EdwardsPoint;
+use curve25519_dalek::scalar::Scalar;
+
+/// Fixed-base scalar multiplication by the Ed25519 base point.
+///
+/// Uses precomputed basepoint tables when the `fast` feature is enabled,
+/// trading off increased code size for better performance.
+pub(crate) fn mul_base(scalar: &Scalar) -> EdwardsPoint {
+    #[cfg(not(feature = "fast"))]
+    {
+        scalar * constants::ED25519_BASEPOINT_POINT
+    }
+
+    #[cfg(feature = "fast")]
+    {
+        scalar * constants::ED25519_BASEPOINT_TABLE
+    }
+}
```

Comment:
Thinking about this a bit more, it seems like this is a concern which could be pushed down into `curve25519-dalek`

Reply:
Moved to https://github.com/dalek-cryptography/curve25519-dalek/pull/489